app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/minigame/FireOutbreakActivity.java

```diff
@@ -168,5 +179,4 @@ public void onPlasmaPress(View view) {
             validateAction(-3);
```

Comment:
Why are you doing this at all? You shouldn't have to send invalid numbers to the server, only send messages when they are valid. If something has to be in order locally with the minigame, just verify it locally, don't bother the server with a message that is always wrong. The purpose of the server is to validate messages that are potentially right. Does this story make sense?